java/runtime/src/main/java/io/ray/runtime/task/NativeTaskSubmitter.java

```diff
@@ -46,7 +46,8 @@ public BaseActorHandle createActor(
       if (options.group != null) {
         PlacementGroupImpl group = (PlacementGroupImpl) options.group;
         Preconditions.checkArgument(
-            options.bundleIndex >= 0 && options.bundleIndex < group.getBundles().size(),
+            options.bundleIndex == -1
```

Comment:
Could you add a comment here to explain why we need filter -1.

Reply:
Ok. Added.